packages/node_modules/@node-red/editor-client/src/js/ui/contextMenu.js

```diff
@@ -44,14 +44,10 @@ RED.contextMenu = (function () {
         const canRemoveFromGroup = hasSelection && !!selection.nodes[0].g
         const offset = $("#red-ui-workspace-chart").offset()
 
-        let addX = options.x - offset.left + $("#red-ui-workspace-chart").scrollLeft()
-        let addY = options.y - offset.top + $("#red-ui-workspace-chart").scrollTop()
-
-        if (RED.view.snapGrid) {
-            const gridSize = RED.view.gridSize()
-            addX = gridSize * Math.floor(addX / gridSize)
-            addY = gridSize * Math.floor(addY / gridSize)
-        }
```

Comment:
Should we not align to grid if user option is set?

Reply:
The code to display the quick-add dialog already handled aligning the provided point to the grid if necessary - so no need to do it twice.